compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -160,6 +160,56 @@ luci::CircleConst *create_NHWC_from_NCHW(luci::CircleConst *constant)
   return nhwc_const;
 }
 
+// We assume MUL is NCHW if,
+// Input shape: (N, C, H, W)
+// Output shape: (N, C, H, W)
+// 1. Const shape is (1, C, 1, 1)
```

Comment:
What the numbers mean?

Reply:
N: batch size
C: channel size
H: height
W: width

This PR only supports a special form of MUL which has the constant with the shape (1, C, 1, 1).

Reply:
Ah, my focus was `1.` and `2.` :)

For me, it is little bit confused because I don't know these are some of conditions or just descriptions.

Reply:
The comment might have confused you. I think it would be better to change it.

How about this?

```cpp
// Find MUL with an NCHW pattern described below
// Input shape: (N, C, H, W)
// Output shape: (N, C, H, W)
// 1. Const shape is (1, C, 1, 1)
// 2. Input, Output, Const have the same C.
// The above condition can be extended later
bool is_NCHW_with_const(const luci::CircleMul *node, luci::CircleNode *&pred_node,
                        luci::CircleConst *&multiplier)
```

Reply:
If I understand correctly, how about following comment? :)
```
// NOTE Following conditions can be extended later
//
// Find MUL with an NCHW pattern described below
//   - Input (non-constant) shape : [N, C, H, W]
//   - Input (constant) shape : [1, C, 1, 1]
//   - Output shape : [N, C, H, W]
```

Reply:
Updated. Thanks.